api/src/groups/api.py

```diff
@@ -24,29 +24,29 @@ def create_group_type():
     try:
         valid_group_type = group_type_schema.load(request.json)
     except ValidationError as err:
-        return jsonify(err.messages), 422
+        return log(jsonify(err.messages), 422)
 
     new_group_type = GroupType(**valid_group_type)
     db.session.add(new_group_type)
     db.session.commit()
-    return jsonify(group_type_schema.dump(new_group_type)), 201
+    return log(jsonify(group_type_schema.dump(new_group_type)), 201)
```

Comment:
Looking through this file, the `log(jsonify())` pattern appears a _lot_. Could we define a new function (maybe `logged` or `logged_response`) that simply took two arguments? The function could inspect the first argument; if it is an object, then invoke `jsonify`.

Reply:
Sure. That makes sense. Should we replace `log` with that maybe?

Reply:
Yes.